qa/rpc-tests/mempool_nu_activation.py

```diff
@@ -67,16 +65,8 @@ def nu_activation_checks():
             # Fill the mempool with twice as many transactions as can fit into blocks
             node0_taddr = self.nodes[0].getnewaddress()
             x_txids = []
-            info = self.nodes[0].getblockchaininfo()
-            chaintip_branchid = info["consensus"]["chaintip"]
             while self.nodes[1].getmempoolinfo()['bytes'] < 2 * 4000:
-                try:
-                    x_txids.append(self.nodes[1].sendtoaddress(node0_taddr, Decimal('0.001')))
-                    assert_equal(chaintip_branchid, "00000000")
-                except JSONRPCException:
-                    # This fails due to expiring soon threshold, which applies from Overwinter onwards.
```

Comment:
I could remove the change to `connect_nodes_bi`. I actually couldn't get this test to run to completion locally (I would crash on one of the sync_all calls with a bad line ending)  and that change seemed to make no difference, but I left it because if felt more "normal".

I believe when introducing the tx expiring soon threshold we broke the ability to send transactions just prior to a network upgrade. Because we would set the expiry height to be `activation height - 1` in the case that tx's expiry would cross a network upgrade, transactions sent on blocks H-1 and H-2 would always be expiring soon. See: https://github.com/zcash/zcash/pull/3689/commits/067b6f9373bbd946db1cd6bb68a4e3760b598410.

In this PR I reenabled the ability to sent transactions at H-1 and H-2 which broke the above change. My fix was to essentially undo the above commit.

Reply:
That all being said, based on discussion, I plan to undo the change to the expiry height near NU activation, and the change to this test.

Reply:
I removed the commit that updated this test and force pushed.